core_java/jdbc/pom.xml

```diff
@@ -5,6 +5,7 @@
   <version>1.0-SNAPSHOT</version>
```

Comment:
please remove all generated files (e.g. .class)

Reply:
@jarviscanada Removed all generated classes. I have re-requested the review.